lang/cs/Org.Apache.REEF.All/packages.config

```diff
@@ -0,0 +1,22 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I don't think we need StyleCop in this project, as it is used only to create the NuGet that includes all of REEF.